package-examples/helloworld-kustomize/README.md

```diff
@@ -0,0 +1,19 @@
+helloworld-kustomize
+==================================================
+
+# NAME
+
+  helloworld-kustomize
+
+# SYNOPSIS
+
+  kubectl apply --recursive -f helloworld-kustomize
```

Comment:
We should use `kpt live apply`

Reply:
I think this one has to use kustomize or kubectl because it has a patch.